src/applications/disability-benefits/686/reducers/authorization.js

```diff
@@ -0,0 +1,44 @@
+import { requestStates } from '../../../../platform/utilities/constants';
+import {
+  LOAD_DISABILITY_RATING_STARTED,
+  LOAD_DISABILITY_RATING_SUCCEEDED,
+  LOAD_DISABILITY_RATING_FAILED
+} from '../actions';
+
+const initialState = {
+  isLoading: false,
+  hasError: false,
+  disabilityRatingState: requestStates.notCalled,
+  payload: null,
+};
+
+function authorizationReducer(state = initialState, action) {
```

Comment:
Should we rename this disabilityRating to distinguish from the authorization selector?